enterprise/com/src/main/java/org/neo4j/com/storecopy/TransactionCommittingResponseUnpacker.java

```diff
@@ -328,9 +420,99 @@ public void obligation( long txId ) throws IOException
+            //      id reuse contamination.
+            // -> Slave promise to not let any transactions cross the safe zone boundary.
+            //      Meaning all transactions that falls out of the safe zone, as updates gets applied,
+            //      will need to be terminated, with a hint that they can simply be restarted
+            //
+            // Safe zone is a snapshotTime frame in Masters domain. All transactions that started and finished within this
+            // snapshotTime frame are guarantied to not have read any mixed state records.
+            //
+            // LARGE ASCII EXAMPLE
+            //
+            //    x---------------------------------------------------------------------------------->| TIME
+            //    |MASTER STATE
+            //    |---------------------------------------------------------------------------------->|
+            //    |                                                          Batch to apply to slave
+            //    |                                                        |<------------------------>|
+            //    |
+            //    |                                                        A
+            //    |SLAVE STATE 1 (before applying batch)         |<---T--->|
+            //    |----------------------------------------------+-------->|
+            //    |                                                        |
+            //    |                                                        |
+            //    |                                                        |      B
+            //    |SLAVE STATE 2 (mid apply)                            |<-+-T--->|
+            //    |-----------------------------------------------------+--+----->|
+            //    |                                                        |      |
+            //    |                                                        |      |
+            //    |                                                        |      |  C
+            //    |SLAVE STATE 3 (mid apply / after apply)                 |<---T-+->|
+            //    |--------------------------------------------------------+------+->|
+            //    |                                                        |      |  |
+            //    |                                                        |      |  |
+            //    |                                                        |      |  |                D
+            //    |SLAVE STATE 4 (after apply)                             |      |  |      |<---T--->|
+            //    |--------------------------------------------------------+------+--+------+-------->|
+            //
+            // * Tx start on slave when slave is in SLAVE STATE 1
+            //      - Latest applied transaction has timestamp A and safe zone is A-T.
+            //      - Tx.startTime = A
```

Comment:
So "startTime" is the time this transaction was committed? Because `A` is at the end of the transaction.
Also, given this information; what does it mean? Can this transaction commit, or not?